src/tests/users.test.ts

```diff
@@ -0,0 +1,207 @@
+import { expect, test } from '@/fixtures/fixtures';
+import Navbar from '@/pages/Navbar';
+import AppConfig from '@/utils/AppConfig';
+
+test.beforeEach(async ({     
+    page,
+ }) => {
+    await page.goto('./dashboard');
+  });
+
+//OBPIH-4622 Users
+test('Create user, add default location and auto-login and delete user', async ({navbar, userListPage, createUserPage, editUserPage, page, loginPage, locationChooser}) => {
+  test.setTimeout(3 * 60 * 1000);
+    await navbar.configurationButton.click();
+    await navbar.getNavItem('Users').click();
+    await userListPage.createUserButton.click();
+    await createUserPage.username.fill('test1');
+    await createUserPage.firstName.fill('test1');
+    await createUserPage.lastName.fill('test1');
+    await createUserPage.password.fill('test1');
```

Comment:
I think we can move this whole part into a separate method on `CreateUserPage` like
```ts
async fillUserForm(data) {
    await this.username.fill(data.username);
    await this.firstName.fill(data.firstName);
    await this.lastName.fill(data.lastName);
    await this.password.fill(data.password);
}

```
and then use it like that in the test
```ts
const formData = {
  username: 'test',
  firstName: 'test',
  lastName: 'test',
  password: 'test'
}

await createUserPage.fillUserForm(formData);
```